### ISSUE TITLE
session preservation fails on start

### ISSUE THREAD

**Issue by stormparticle** [stormparticle/spoticli#29]
while upgrading to tekore 3.0.1, ran into this:
only occurred on WSL, not on win10

```
mars:~$ ./spot.sh
cannot read auth.spoticli
attempting to delete auth.spoticli
auth.spoticli deleted
please restart spoticli

mars:~$ .//spot.sh
Opening browser for Spotify login...
if no browser opens manually navigate to URL below:
[omitted]
warning, failed to write token! session will not be preserved!
```

doing a debug this is the error for why session preservation fails:
```
Traceback (most recent call last):
  File "/mnt/c/Users/hlopez/Projects/spoticli/__init__.py", line 123, in <module>
    pickle.dump(spot_token, open('auth.spoticli', 'wb'))
TypeError: can't pickle _thread.RLock objects
```

**Comment by stormparticle** [comment]
replicated on my laptop (native debian 10)
replicated on win10 (i had forgotten to upgrade to tekore 3.0.1 before troubleshooting)
issue appears to be due to upgrade to tekore 3.0.1

**Comment by stormparticle** [comment]
not sure why this is the case, don't really feel like digging into tekore source to figure out what changes.
issue is fixed now in commit 7ad459af83f8ca614693e8d9734bff21f55951bc
